app/Http/Controllers/Adm/Dashboard.php

```diff
@@ -76,7 +30,7 @@ public function anySearch($searchQuery = null)
         }
 
         if (strlen($searchQuery) < 2 or $searchQuery == null) {
```

Comment:
`strlen` should accept the null value, so you can just write `strlen($searchquery) < 2`

Reply:
This PR is the start of us decommissioning the admin panel so this code will soon be deleted and replaced with Nova 😄